docs/how-to/default-profile.md

```diff
@@ -0,0 +1,96 @@
+
+Your IPFS profile is found within your node's `config` file. The default location for the `config` file is `~/.ipfs/config`. If you have set an `$IPFS_PATH` variable, you can find your `config` file at `$IPFS_PATH/config`. Use `grep` to find the currently set profile:
+
+```bash
+cat ~/.ipfs/config | grep "prefix"
+# or
+cat $IPFS_PATH/config | grep "prefix"
+
+> "prefix": "flatfs.datastore",
+> "prefix": "leveldb.datastore",
+```
+
+IPFS uses the `flatfs` profile by default. `flatfs` is implemented using LevelDB, which is why you also see another line in the output with `leveldb.datastore`, but they both refer to `flatfs` datastore.
+
+If previously you configured your IPFS node to use another profile, let's say `badgerds`, the above command would give you an output similar to the following:
+
+```bash
+"prefix": "badger.datastore",
+```
+
+This output line refers to `badgerds` profile .
+
+## Available profiles
+
+Here's a list of all the profiles available for your IPFS node. You can compare the output from above command with any of the following profiles to find what profile is currently being used by your node:
+
+- `flatfs`
+- `badgerds`
+- `server`
+- `randomports`
+- `default-datastore`
+- `local-discovery`
+- `test`
+- `default-networking`
+- `lowpower`
+
+## Reset your profile
+
+If you want to reset your IPFS node profile to the default profile, you can do so by using the following method:
+
+```diff
+ipfs config profile apply default-datastore
+
+...
+"Datastore": {
+    "BloomFilterSize": 0,
+    "GCPeriod": "1h",
+    "HashOnRead": false,
+    "Spec": {
+-       >> "child": {"path":"badgerds","syncWrites":false,"truncate":true,"type":"badgerds"},
++       << "mounts": [{"child":{"path":"blocks","shardFunc":"/repo/flatfs/shard/v1/next-to-last/2","sync":true,"type":"flatfs"},"mountpoint":"/blocks","prefix":"flatfs.datastore","type":"measure"},{"child":{"compression":"none","path":"datastore","type":"levelds"},"mountpoint":"/","prefix":"leveldb.datastore","type":"measure"}],
+-       >> "prefix": "badger.datastore",
+        <> "type": "measure"
+        ** "type": "mount"
+    },
+    "StorageGCWatermark": 90,
+    "StorageMax": "10GB"
+},
+...
+```
+
+Above command shows the difference between your existing IPFS config and the new configuration. Also, it would overwrite your IPFS `config` file, so it's always better to create a backup of your existing IPFS configuration before running this command.
+
+## Converting profiles
+
+Not all profiles are compatible with each other simply because they use very different technologies for storing the data inside the datastores. For instance, if you want to convert `badgerds` to `default-datastore`, you would have to use another helper tool called [ipfs-ds-convert](https://dist.ipfs.io/#ipfs-ds-convert) to convert the datastore to the required format.
+
+Install `ipfs-ds-convert` tool from the given link and run the following command to finally convert the datastore:
```

Comment:
Tell the user how to _install_ this tool. It looks like they just have to download it, unzip the package, and run the executable.

Reply:
sure @johnnymatthews i'll add instructions about installing `ipfs-ds-convert` as well